demo/load-tests/test-load.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
So are we not allowed to use the new, shorter license headers for these JS files?

Reply:
We would be, yes, but since they are not part of the deployed bundle, it doesn't matter how long the header is.  I thought it would be best to keep the headers explicit everywhere but index.js.

But honestly, this policy is new to me, so I'm just guessing and making judgement calls based on what feels right to me.